src/main/java/core/basesyntax/figure/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figure;
+
+public class Square extends AbstractFigure implements Figure {
```

Comment:
The class 'Square' should not implement 'Figure' here since it is already extending 'AbstractFigure' which I assume implements 'Figure'. This is redundant and can be removed.